implementation/jobberknoll/api/deno.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@jobberknoll/api",
-  "version": "1.0.0",
```

Comment:
Consolidating separate versions for subpackages in the workspace to a single workspace version.